src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.html

```diff
@@ -112,7 +112,7 @@ <h4 class="sb-pageSection-title m-0 mr-5"> {{contentTitle}}</h4>
                                 </div>
                                 <div class="ui grid">
                                     <div class="twelve wide column" [appTelemetryImpression]="telemetryContentImpression">
-                                        <app-player [contentProgressEvents$]="contentProgressEvents$" (assessmentEvents)="assessmentEvents($event)" (questionScoreSubmitEvents)="questionScoreSubmitEvents($event)" class="content-player" [playerConfig] ="playerConfig" [playerOption] = "playerOption"></app-player>
+                                        <app-player [telemetryObject]="contentInteractObject" [pageId]="pageId" [overlayImagePath]="overlayImagePath" [contentProgressEvents$]="contentProgressEvents$" (assessmentEvents)="assessmentEvents($event)" (questionScoreSubmitEvents)="questionScoreSubmitEvents($event)" class="content-player" [playerConfig] ="playerConfig" [playerOption] = "playerOption"></app-player>
```

Comment:
Why do you want to pass this **overlayImagePath**, It can handle in app-player component itself.
This should be same at all places. To maintain consistance.

Reply:
IT should take it from contentData which is passing as input to app-player

Reply:
contentData won't have pagId, and as per mobile, the background image is appIcon of the content, not any image (different for all)